woocommerce/woocommerce-android
# Feature/7510 add login access

Closes: #7510 


### Description
This PR adds 2 changes: 
- Kick off store creation flow after sign up.
- Handling “Log in” click inside `SignUpFragment` to open login screen with WP.com credentials. 

### Testing instructions
**Test 1**

- Click on "Log in" and check the WP.com login screen is opened.

**Test 2**

- Create a new account
- Check that the store creation web flow is triggered.
- Navigate back and check site picker is displayed with store creation CTA visible. 


### Images/gif

https://user-images.githubusercontent.com/2663464/198369122-cf58edda-266a-4442-b440-ee3277c4c236.mp4
